torch/testing/_internal/common_methods_invocations.py

```diff
@@ -10005,17 +10012,27 @@ def ref_pairwise_distance(input1, input2):
     OpInfo('quantile',
            dtypes=floating_types(),
            sample_inputs_func=sample_inputs_reduction_quantile,
+           supports_forward_ad=True,
            skips=(
                # Pre-existing condition; Needs to be fixed
                DecorateInfo(unittest.expectedFailure, 'TestCommon', 'test_composite_compliance'),
+               # This is a composite op that `sort` enables, but got: Trying to set a forward gradient
```

Comment:
Is that because the sort formula is wrong?

Reply:
I think it was actually failing because of the batched forward grad check